src/service/git/git-token-service.ts

```diff
@@ -0,0 +1,101 @@
+import { Service } from "typedi";
+
+@Service()
+export class GitTokenService {
+  private githubTokens: Record<string, string>;
+  private gitlabTokens: Record<string, string>;
+  private githubTokenPools: Record<string, string[]>;
+
+  constructor() {
+    this.githubTokens = {};
+    this.gitlabTokens = {};
```

Comment:
@Ginxo @lampajr For `git-token-service` I just realized that I might not need to keep track github and gitlab tokens separately since platform id should be unique irrespective of the platform. Wdyt? I should merge these maps right?

Reply:
I agree, if we can assure that platform ids are always unique I think merging those maps is the good way. One thing: do we have some checks that prevent multiple platform to have the same ids, I mean uniqueness is guaranteed?

Reply:
I do agree